### ISSUE TITLE
How many CFF files are valid?

### ISSUE THREAD

**Issue by sdruskat** [sdruskat/cff-in-the-wild#7]
This could be done by reusing [cffconvert](https://github.com/citation-file-format/cff-converter-python/) to validate the files (and record error messages).

**Comment by Bisaloo** [comment]
I'm going to use https://docs.ropensci.org/jsonvalidate/, which gives clear error messages.

It could still be useful to compare with the output of another validator though. Hopefully, the results should be the same.

**Comment by amal-ghamdi** [comment]
I can add myself in visualization

![pie_1](https://user-images.githubusercontent.com/746326/162183511-25d4ff1f-000d-44ea-bf5c-69de8bf9e6c6.png)

**Comment by amal-ghamdi** [comment]
some updates
![validity](https://user-images.githubusercontent.com/746326/162185594-c1f54b86-e90c-4837-b730-9e209d0be91a.png)

**Comment by Bisaloo** [comment]
I think we have two levels of 'invalidity': 

- the yaml file cannot be parsed
- the yaml file can be parsed but is not a valid CFF file

@amal-ghamdi, what do you think of having two hues of 'invalid' to distinguish between these two types?

**Comment by amal-ghamdi** [comment]
yes makes sense, this should also be reflected in the data @samharrison7  provides as well

**Comment by Bisaloo** [comment]
On my end, out of 3307 files, I can parse 3146 and 1804 of them are valid CFF files. I can provide a script later if these results differ from what @samharrison7 finds :slightly_smiling_face:

**Comment by amal-ghamdi** [comment]
Got it!
